app/main/mainController.js

```diff
@@ -101,4 +130,35 @@
             }
         })();
     });
+
+    function PanelMenuCtrl (mdPanelRef, InstitutionService, $state, $timeout) {
+        var panelCtrl = this;
+        panelCtrl._mdPanelRef = mdPanelRef;
+
+        $timeout(function() {
+            var selected = document.querySelector('.demo-menu-item.selected');
+            if (selected) {
+              angular.element(selected).focus();
+            } else {
+              angular.element(document.querySelectorAll('.demo-menu-item')[0]).focus();
+            }
+          });
+
+        panelCtrl.goToSearchedInstitution = function goToSearchedInstitution(institutionId) {
```

Comment:
Daria, tava assim antes, mas fiz isso para poder testar. Isso porque o mdPanelRef só existe quando o main controller instancia o panel com o $mdPanel, como nos testes eu to testando o controller separado, não tem como eu ter o mdPanelRef e assim não posso chamar funções que usam ele. Assim, para poder testar o o state.go e o método do service eu tive que modularizar dessa forma.